src/style.css

```diff
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
```

Comment:
Don't hardcode width and height for main container

Reply:
Fixed